Using CAPSTONE_X86_REDUCE breaks the build

I am using the latest commit on the next branch and have enabled `CAPSTONE_X86_SUPPORT` and `CAPSTONE_X86_REDUCE`. But now I get the following errors:
```
[9/29] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DCAPSTONE_BPF_SUPPORT -DCAPSTONE_HAS_BPF -DCAPSTONE_HAS_MOS65XX -DCAPSTONE_HAS_RISCV -DCAPSTONE_HAS_WASM -DCAPSTONE_HAS_X86 -DCAPSTONE_MOS65XX_SUPPORT -DCAPSTONE_RISCV_SUPPORT -DCAPSTONE_USE_SYS_DYN_MEM -DCAPSTONE_WASM_SUPPORT -DCAPSTONE_X86_REDUCE -DCAPSTONE_X86_SUPPORT -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.1.sdk -std=gnu99 -MD -MT CMakeFiles/capstone.dir/arch/X86/X86IntelInstPrinter.c.o -MF CMakeFiles/capstone.dir/arch/X86/X86IntelInstPrinter.c.o.d -o CMakeFiles/capstone.dir/arch/X86/X86IntelInstPrinter.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c
FAILED: CMakeFiles/capstone.dir/arch/X86/X86IntelInstPrinter.c.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DCAPSTONE_BPF_SUPPORT -DCAPSTONE_HAS_BPF -DCAPSTONE_HAS_MOS65XX -DCAPSTONE_HAS_RISCV -DCAPSTONE_HAS_WASM -DCAPSTONE_HAS_X86 -DCAPSTONE_MOS65XX_SUPPORT -DCAPSTONE_RISCV_SUPPORT -DCAPSTONE_USE_SYS_DYN_MEM -DCAPSTONE_WASM_SUPPORT -DCAPSTONE_X86_REDUCE -DCAPSTONE_X86_SUPPORT -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.1.sdk -std=gnu99 -MD -MT CMakeFiles/capstone.dir/arch/X86/X86IntelInstPrinter.c.o -MF CMakeFiles/capstone.dir/arch/X86/X86IntelInstPrinter.c.o.d -o CMakeFiles/capstone.dir/arch/X86/X86IntelInstPrinter.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c:1038:8: error: use of undeclared identifier 'X86_BNDCL32rm'
                case X86_BNDCL32rm:
                     ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c:1039:8: error: use of undeclared identifier 'X86_BNDCN32rm'; did you mean 'X86_ANDN32rm'?
                case X86_BNDCN32rm:
                     ^~~~~~~~~~~~~
                     X86_ANDN32rm
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86GenInstrInfo_reduce.inc:136:2: note: 'X86_ANDN32rm' declared here
        X86_ANDN32rm    = 246,
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c:1040:8: error: use of undeclared identifier 'X86_BNDCU32rm'
                case X86_BNDCU32rm:
                     ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c:1041:8: error: use of undeclared identifier 'X86_BNDSTXmr'
                case X86_BNDSTXmr:
                     ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c:1042:8: error: use of undeclared identifier 'X86_BNDLDXrm'
                case X86_BNDLDXrm:
                     ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c:1043:8: error: use of undeclared identifier 'X86_BNDCL64rm'
                case X86_BNDCL64rm:
                     ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c:1044:8: error: use of undeclared identifier 'X86_BNDCN64rm'; did you mean 'X86_ANDN64rm'?
                case X86_BNDCN64rm:
                     ^~~~~~~~~~~~~
                     X86_ANDN64rm
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86GenInstrInfo_reduce.inc:138:2: note: 'X86_ANDN64rm' declared here
        X86_ANDN64rm    = 248,
        ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/capstone/src/77d43ad0c6-c7ea840cfd.clean/arch/X86/X86IntelInstPrinter.c:1045:8: error: use of undeclared identifier 'X86_BNDCU64rm'
                case X86_BNDCU64rm:
                     ^
8 errors generated.
```

--- a/arch/X86/X86ATTInstPrinter.c
+++ b/arch/X86/X86ATTInstPrinter.c
@@ -801,6 +801,7 @@ static void printanymem(MCInst *MI, unsigned OpNo, SStream *O)
 		case X86_LEA64r:
 				 MI->x86opsize = 8;
 				 break;
+#ifndef CAPSTONE_X86_REDUCE
 		case X86_BNDCL32rm:
 		case X86_BNDCN32rm:
 		case X86_BNDCU32rm:
@@ -811,6 +812,7 @@ static void printanymem(MCInst *MI, unsigned OpNo, SStream *O)
 		case X86_BNDCU64rm:
 				 MI->x86opsize = 16;
 				 break;
+#endif
 	}
 
 	printMemReference(MI, OpNo, O);

